modules/oe_content_reference/config/install/field.storage.node.oe_reference.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.oe_reference
```

Comment:
Rename this field into `oe_reference_code`, so to avoid confusion with the other meaning of the word reference in Drupal.

Reply:
done